src/components/learner-credit-management/BudgetDetailPageOverviewAvailability.jsx

```diff
@@ -95,6 +95,46 @@ const BudgetActions = ({
     );
   }
 
+  if (status === BUDGET_STATUSES.retired) {
+    return (
+      <div className="h-100 d-flex align-items-center pt-4 pt-lg-0">
+        <div>
+          <h3>
+            <FormattedMessage
+              id="lcm.budget.detail.page.overview.budget.actions.keep.people.learning.active.budget"
+              defaultMessage="Keep people learning with an active budget"
+              description="Title for the budget actions section on the budget detail page overview"
+            />
+          </h3>
+          <p>
+            <FormattedMessage
+              id="lcm.budget.detail.page.overview.budget.actions.budget.retired"
+              defaultMessage="This budget is retired. To create a new budget, please contact your representative."
```

Comment:
nit: to be consistent with the other "representative" -> "support" changes...

```suggestion
              defaultMessage="This budget is retired. To create a new budget, please contact support."
```